<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rober</class>
 <widget class="QMainWindow" name="Rober">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>로버</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(47, 47, 47);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="conButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(40, 74, 102);
font: 75 14pt &quot;Ubuntu Mono&quot;;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="statusText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusTxt_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statusTxt_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Ubuntu Mono&quot;;
background-color: rgb(40, 74, 102);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>R</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Ubuntu Mono&quot;;
background-color: rgb(40, 74, 102);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="speedText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Ubuntu Mono&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;Ubuntu Mono&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>13</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 13pt &quot;Ubuntu Mono&quot;;
background-color: rgb(40, 74, 102);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;Ubuntu Mono&quot;;
background-color: rgb(40, 74, 102);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel0">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="locLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
